### PR TITLE
Correct the `transport/storage` namespace for the doc and remove the scripts

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

Refer: https://issues.redhat.com/browse/ACM-7543

### CODE REVIEW

File: doc/troubleshooting.md [hunk 1]
```diff
@@ -9,52 +9,52 @@ Depending on the type of service, there are three ways to access the [provisione
 * `ClusterIP` service
     1. Run the following command to determine your postgres connection URI:
         ```
-        kubectl get secrets -n multicluster-global-hub-postgres hoh-pguser-postgres -o go-template='{{index (.data) "uri" | base64decode}}'
+        kubectl get secrets -n multicluster-global-hub postgres-pguser-postgres -o go-template='{{index (.data) "uri" | base64decode}}'
         ```
     2. Run the following command to access the database:
         ```
-        kubectl exec -it $(kubectl get pods -n multicluster-global-hub-postgres -l postgres-operator.crunchydata.com/role=master -o jsonpath='{.items..metadata.name}') -c database -n multicluster-global-hub-postgres -- psql -U postgres -d hoh -c "SELECT 1"
+        kubectl exec -t $(kubectl get pods -n multicluster-global-hub -l postgres-operator.crunchydata.com/role=master -o jsonpath='{.items..metadata.name}') -c database -n multicluster-global-hub -- psql -U postgres -d hoh -c "SELECT 1"
```
Comment (morvencao) [inline doc/troubleshooting.md:16]: `-t` or `-it`?

File: operator/config/samples/storage/postgres-subscription.yaml [hunk 2]
```diff
@@ -1,17 +1,8 @@
-apiVersion: operators.coreos.com/v1
-kind: OperatorGroup
-metadata:
-  name: crunchy-postgres-operator
-  namespace: multicluster-global-hub-postgres
-spec:
-  targetNamespaces:
-    - multicluster-global-hub-postgres
----
```
Comment (morvencao) [inline operator/config/samples/storage/postgres-subscription.yaml:9]: there is operatorGroup CR in `multicluster-global-hub` namespace, right?

File: doc/troubleshooting.md (hunk 1 above)
Comment (yanmxa) [reply doc/troubleshooting.md:16]: `-t`

File: operator/config/samples/storage/postgres-subscription.yaml (hunk 2 above)
Comment (yanmxa) [reply operator/config/samples/storage/postgres-subscription.yaml:9]: Yes, It has the default operatorgroup.

File: doc/troubleshooting.md (hunk 1 above)
Comment (yanmxa) [reply doc/troubleshooting.md:16]: Done.

**Comment by yanmxa** [comment]
wait the confirmation from @clyang82

File: operator/config/samples/storage/postgres-cluster.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 apiVersion: postgres-operator.crunchydata.com/v1beta1
 kind: PostgresCluster
 metadata:
-  name: hoh
-  namespace: multicluster-global-hub-postgres
+  name: postgres
+  namespace: multicluster-global-hub
```
Comment (clyang82) [inline operator/config/samples/storage/postgres-cluster.yaml:5]: It is only used for BYO case, maybe a separated namespace makes more sense. If it is in `multicluster-global-hub`, why not use the operator to do it. Any thoughts?

File: operator/README.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ The operator of multicluster global hub (see: https://github.com/stolostron/mult
 kubectl create secret generic multicluster-global-hub-storage -n "multicluster-global-hub" \
     --from-literal=database_uri=<postgresql-uri> 
 ```
-> You can run this sample script `config/samples/storage/deploy_postgres.sh` to install postgres in `multicluster-global-hub-postgres` namespace and create the secret `multicluster-global-hub-storage` in namespace `multicluster-global-hub` automatically. To override the secret namespace, set `TARGET_NAMESPACE` environment variable to the ACM installation namespace before executing the script.
+> You can run this sample script `config/samples/storage/deploy_postgres.sh` to install postgres in `multicluster-global-hub` namespace and create the secret `multicluster-global-hub-storage` in namespace `multicluster-global-hub` automatically. To override the secret namespace, set `TARGET_NAMESPACE` environment variable to the ACM installation namespace before executing the script.
```
[the remark this answers is not in the document]
Comment (yanmxa) [reply operator/README.md]: Done.

File: doc/byo.md [hunk 5]
```diff
@@ -95,9 +95,9 @@ Notes:
 ```
 into `PostgresCluster` operand and then you can get the EXTERNAL-IP from `hoh-ha` service. for example:
```
[the remark this answers is not in the document]
Comment (yanmxa) [reply doc/byo.md]: Done.

File: operator/config/samples/storage/postgres-cluster.yaml (hunk 3 above)
Comment (yanmxa) [reply operator/config/samples/storage/postgres-cluster.yaml:5]: As we discussed. We will remove the scripts from here.

File: operator/README.md [hunk 6]
```diff
@@ -2,27 +2,6 @@
 
 The operator of multicluster global hub (see: https://github.com/stolostron/multicluster-global-hub)
 
-## Prerequisites
-
-1. Connect to a Kubernetes cluster with `kubectl`
-2. ACM or OCM is installed on the Kubernetes cluster
-3. PostgreSQL is installed and a database is created for multicluster global hub, also a secret with name `multicluster-global-hub-storage` that contains the credential should be created in `multicluster-global-hub` namespace. The credential format like `postgres://<user>:<password>@<host>:<port>/<database>`:
```
Comment (clyang82) [inline operator/README.md:9]: we need `#1` and `#2` as Prerequisites

Comment (yanmxa) [reply operator/README.md:9]: Done.

**Pull request merged**